airbyte-integrations/connectors/source-postgres/3-run-script.sql

```diff
@@ -0,0 +1,142 @@
+create sequence test_seq;
+
+create table test(
+id int check (id > 0) not null default nextval ('test_seq') primary key,
+varchar1 varchar(255),
+varchar2 varchar(255),
+varchar3 varchar(255),
+varchar4 varchar(255),
+varchar5 varchar(255),
+longblobfield bytea,
+timestampfield timestamp(0));
+
+-- TODO: change the following @allrows to control the number of records with different sizes
+-- number of 50B records
+call insert_rows(0, 500000, '''test weight 50b - 50b text, 50b text, 50b text''');
+-- number of 500B records
+call insert_rows(0, 50000, '''test weight 500b - 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text''');
```

Comment:
For Postgres, there is also a [repeat](https://www.postgresql.org/docs/9.1/functions-string.html) function you can use.

Reply:
@tuliren, thanks, those are great comments. I corrected the script and changed it so that the insert adds the exact number of records.